spec/unit/ssl/oids_spec.rb

```diff
@@ -34,6 +34,9 @@
       'pp_apptier'          => "1.3.6.1.4.1.34380.1.1.24",
       'pp_hostname'         => "1.3.6.1.4.1.34380.1.1.25",
       'ppPrivCertExt'       => '1.3.6.1.4.1.34380.1.2',
+      'ppAuthCertExt'       =>  '1.3.6.1.4.1.34380.1.3',
+      'pp_authorization'    =>  '1.3.6.1.4.1.34380.1.3.1',
+      'pp_auth_role'        =>  '1.3.6.1.4.1.34380.1.3.13',
```

Comment:
Nit: let's keep these whitespace aligned with the entries above. It's slightly easier on my eyes :)

And nit unrelated to this PR, but the mix of double-quotes and single-quotes in the entries above has no rhyme or reason here AFAICT.